kratos/python_scripts/process_factory.py

```diff
@@ -15,35 +15,7 @@ def ConstructListOfProcesses( self, process_list ):
                 python_module = __import__(item["python_module"].GetString())
                 p = python_module.Factory(item, self.Model)
                 constructed_processes.append( p )
-            elif(item.Has("implemented_in_module")):
-                print("************************************************************************")
-                print("******** WARNING USING THE OLD INTERFACE *******************************")
-                print("****** kratos_module or python_module should be prescribed instead of **")
-                print("****** implemented_in_module and implemented_in_file                  **")
-                print("****** old interface is deprecated and will be removed                **")
-                print("************************************************************************")
-                module = __import__(item["implemented_in_module"].GetString())
-                interface_file = __import__(item["implemented_in_file"].GetString())
-                p = interface_file.Factory(item, self.Model)
-                constructed_processes.append( p )
-
-            #if( "implemented_in_python" in item): #check if implemented in python or in c++
-                #if item["implemented_in_python"] == True: #here we treat the case of python implemented processes
-                    #kratos_module = __import__(item["kratos_module"])
-                    #python_module = __import__(item["python_module"])
-                    #p = python_module.Factory(item, self.Model)
-                    #constructed_processes.append( p )
-                    
-                #else: #here we create c++ processes
-                    #kratos_module = __import__(item["kratos_module"])
-                    #python_module = __import__(item["python_module"])
-                    #p = python_module.Factory(item, self.Model)
-                    #constructed_processes.append( p )
-                    
-            #else:
-                #raise Exception( "process defined by settings ", item, " does not define if implemented in python or in c++")
-            
-    
+                
```

Comment:
why not throwing a proper error message?
Saying that one has to use the new processes?

Reply:
Any suggestion?, what I am doing here is remove unused code, that was added for the process/json transition more than 2 years ago